components/button/variants.tsx

```diff
@@ -0,0 +1,101 @@
+import { cva } from "class-variance-authority"
+
+export const ButtonVariants = cva(
+  "inline-flex items-center justify-center flex-shrink-0 whitespace-nowrap w-auto rounded-full text-xl transition-colors focus-visible:outline-none disabled:cursor-not-allowed disabled:opacity-50 [&_svg]:pointer-events-none [&_svg]:h-4 [&_svg]:w-4",
+  {
+    variants: {
+      variant: {
+        // ————————————————————— Primary Filled —————————————————————
+        primary_filled: [
+          "bg-blue-cbc text-white shadow-sm",
+          "hover:bg-blue-300",
+          "focus-visible:ring-2 focus-visible:ring-blue-500",
+          "active:bg-blue-500",
+        ].join(" "),
+        // ————————————————————— Secondary Filled ————————————————————
+        secondary_filled: [
+          "bg-sunglow-400 text-black shadow-sm",
+          "hover:bg-sunglow-200",
+          "focus-visible:ring-2 focus-visible:ring-sunglow-300",
+          "active:bg-sunglow-300",
+        ].join(" "),
+        // ————————————————————— Primary Outlined ————————————————————
+        primary_outlined: [
+          "bg-transparent border-2 border-blue-cbc text-blue-cbc",
+          "hover:border-blue-300 hover:text-blue-300",
```

Comment:
CCV website has the equivalent of `hover:bg-blue-300` for more visually accessible hover behavior. same for the secondary_outlined button.

Reply:
Sorry, I don't know what you mean by this?

Reply:
when you hover over the red outlined Work With Us button in the top banner, the button turns red (hover:bg-red-university). desired hover behavior below:
<img width="182" height="68" alt="Screenshot 2025-08-27 at 6 05 21 PM" src="https://github.com/user-attachments/assets/56bae2d4-b944-4b6b-b0a0-fee1a40108f4" />

when you hover over the View Events button in the home page hero, the button stays transparent but the outline color changes to a lighter sunglow. i would change it so the hover behaves like the Work With Us button. The current hover behavior is below: 
<img width="360" height="88" alt="Screenshot 2025-08-27 at 6 05 16 PM" src="https://github.com/user-attachments/assets/1242be26-cd6a-4d34-b14b-d46b260dff80" />